src/content/docs/integrations/kubernetes-integration/installation/kubernetes-integration-install-configure.mdx

```diff
@@ -135,448 +130,24 @@ Before starting our [automated installer](https://one.newrelic.com/launcher/k8s-
-    </Callout>
-  </Collapser>
-
-  <Collapser
-    id="kube-state-metrics-pod-label-discovery"
-    title={<>Discover <InlineCode>kube-state-metrics</InlineCode> pods using a label</>}
-  >
-    If several instances of `kube-state-metrics` are present in the cluster, another option to easily target one of these instances with the Kubernetes integration is to use label-based discovery.
-
-    ```
-    - name: "KUBE_STATE_METRICS_POD_LABEL"
-       value: "<var>LABEL_NAME</var>"
-    ```
-
-    <Callout variant="important">
-      When a `KUBE_STATE_METRICS_POD_LABEL` is defined, the label should have a value equal to `true`. For example, if the label name is `my-ksm`, ensure that `my-ksm=true`.
-    </Callout>
-
-    <Callout variant="important">
-      This configuration option is incompatible with `KUBE_STATE_METRICS_URL`. If you have both defined, `KUBE_STATE_METRICS_URL` is used.
-    </Callout>
-  </Collapser>
-
-  <Collapser
-    id="kube-state-metrics-behind-rbac"
-    title={<>Query <InlineCode>kube-state-metrics</InlineCode> behind RBAC</>}
-  >
-    If your instance of `kube-state-metrics` is behind [kube-rbac-proxy](https://github.com/brancz/kube-rbac-proxy), the integration can be configured in a compatible way using the combination of the label-based discovery and two other environment variables:
-
-    ```
-    - name: "KUBE_STATE_METRICS_SCHEME"
-       value: "https"
-     - name: "KUBE_STATE_METRICS_PORT"
-       value: "<var>KSM_RBAC_PROXY_PORT</var>"
-    ```
-
-    To confirm which port should be used as the value of `KUBE_STATE_METRICS_PORT`, we recommend running a describe command on the `kube-state-metrics` pod and look for the port exposed by the container named `kube-rbac-proxy-main`.
-
-    <Callout variant="important">
-      These two configuration options only work when using the `KUBE_STATE_METRICS_POD_LABEL` configuration described above.
-    </Callout>
-  </Collapser>
-
-  <Collapser
-    id="kube-state-metrics-timeout-change"
-    title={<><InlineCode>kube-state-metrics</InlineCode> timeout: Increase the client timeout</>}
-  >
-    To increase the client timeout of `kube-state-metrics`, add a new environment variable, `TIMEOUT`, to the manifest file:
-
-    ```
-    env:
-      - name: TIMEOUT
-        value: 5000 # The default client timeout when calling kube-state-metrics, in milliseconds
-    ```
-
-    Then, add this new environment variable to the [`NRIA_PASSTHROUGH_ENVIRONMENT`](/docs/infrastructure/install-configure-manage-infrastructure/configuration/infrastructure-configuration-settings)
-  </Collapser>
-
-  <Collapser
-    id="non-default-namespace"
-    title="Non-default namespace deployments: Edit config file"
-  >
-    If you want to deploy in a different namespace from `default`, change all values of `namespace` in the manifest.
-  </Collapser>
-
-  <Collapser
```

Comment:
all the following config options are already described somewhere else?

Reply:
IIRC they should be exposed in the chart `values.yml`